site/content/docs/v1.5/api-types/schedule.md

```diff
@@ -125,6 +125,9 @@ spec:
           # processed. Only "exec" hooks are supported.
           post:
             # Same content as pre above.
+    # Specifies whether to use OwnerReferences on backups created by this Schedule. 
```

Comment:
The change only exists on Velero 1.6+: https://github.com/vmware-tanzu/velero/commit/309d3dcf0afefc900715020031d4dcbfb61d073b

Reply:
ok, I will remove from the v1.5 doc.